Changes in module 'nf-core/multiqc'
--- modules/nf-core/multiqc/main.nf
+++ modules/nf-core/multiqc/main.nf
@@ -11,6 +11,9 @@
     path(multiqc_config)
     path(extra_multiqc_config)
     path(multiqc_logo)
+    val(run_title)
+    val(multiqc_commentary)
+    val(client_information)
 
     output:
     path "*multiqc_report.html", emit: report
@@ -26,6 +29,8 @@
     def config = multiqc_config ? "--config $multiqc_config" : ''
     def extra_config = extra_multiqc_config ? "--config $extra_multiqc_config" : ''
     def logo = multiqc_logo ? /--cl-config 'custom_logo: "${multiqc_logo}"'/ : ''
+    def run_title = "--title $run_title" ?: ''
+    def multiqc_commentary = "--comment \"$client_information || $multiqc_commentary\"" ?: ''
     """
     multiqc \\
         --force \\
@@ -33,6 +38,8 @@
         $config \\
         $extra_config \\
         $logo \\
+        $run_title \\
+        $multiqc_commentary \\
         .
 
     cat <<-END_VERSIONS > versions.yml

************************************************************
