Changes in module 'nf-core/kraken2/kraken2'
--- modules/nf-core/kraken2/kraken2/main.nf
+++ modules/nf-core/kraken2/kraken2/main.nf
@@ -1,6 +1,7 @@
 process KRAKEN2_KRAKEN2 {
     tag "$meta.id"
-    label 'process_high'
+    label 'kraken2'
+    debug true
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
@@ -9,7 +10,7 @@
 
     input:
     tuple val(meta), path(reads)
-    path  db
+    path db
     val save_output_fastqs
     val save_reads_assignment
 
@@ -18,6 +19,7 @@
     tuple val(meta), path('*.unclassified{.,_}*')   , optional:true, emit: unclassified_reads_fastq
     tuple val(meta), path('*classifiedreads.txt')   , optional:true, emit: classified_reads_assignment
     tuple val(meta), path('*report.txt')                           , emit: report
+    tuple val(meta), path('*kraken2.krona')                        , emit: krona
     path "versions.yml"                                            , emit: versions
 
     when:
@@ -33,10 +35,12 @@
     def unclassified_option = save_output_fastqs ? "--unclassified-out ${unclassified}" : ""
     def readclassification_option = save_reads_assignment ? "--output ${prefix}.kraken2.classifiedreads.txt" : "--output /dev/null"
     def compress_reads_command = save_output_fastqs ? "pigz -p $task.cpus *.fastq" : ""
-
+    //def rlength = params.ontreads ? 250 : params.readlen // and here ontreads matters. Default for -r is 100 in bracken, Dilthey used 1k in his paper
+    
     """
     kraken2 \\
         --db $db \\
+        --memory-mapping \\
         --threads $task.cpus \\
         --report ${prefix}.kraken2.report.txt \\
         --gzip-compressed \\
@@ -47,8 +51,20 @@
         $args \\
         $reads
 
+    cut -f 2,3 ${prefix}.kraken2.report.txt > ${prefix}_kraken2.krona
+
     $compress_reads_command
 
+ 
+    touch ${prefix}.kraken2.report.txt
+    if [ "$save_output_fastqs" == "true" ]; then
+        touch $classified
+        touch $unclassified
+    fi
+    if [ "$save_reads_assignment" == "true" ]; then
+        touch ${prefix}.kraken2.classifiedreads.txt
+    fi
+    
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         kraken2: \$(echo \$(kraken2 --version 2>&1) | sed 's/^.*Kraken version //; s/ .*\$//')
@@ -74,12 +90,11 @@
     if [ "$save_reads_assignment" == "true" ]; then
         touch ${prefix}.kraken2.classifiedreads.txt
     fi
-
+    
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         kraken2: \$(echo \$(kraken2 --version 2>&1) | sed 's/^.*Kraken version //; s/ .*\$//')
         pigz: \$( pigz --version 2>&1 | sed 's/pigz //g' )
     END_VERSIONS
     """
-
 }

************************************************************
